sechub-wrapper-xray/src/test/java/com/mercedesbenz/sechub/wrapper/xray/cli/XrayWrapperCommandLineParserTest.java

```diff
@@ -0,0 +1,60 @@
+package com.mercedesbenz.sechub.wrapper.xray.cli;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.mercedesbenz.sechub.wrapper.xray.XrayWrapperException;
+
+class XrayWrapperCommandLineParserTest {
+
+    XrayWrapperCommandLineParser parserToTest;
+
+    @BeforeEach
+    void beforeEach() {
+        parserToTest = new XrayWrapperCommandLineParser();
+    }
+
+    @Test
+    void parseCommandLineArgs_valid_arguments() throws XrayWrapperCommandLineParserException, XrayWrapperException {
+        /* prepare */
+        String[] args = { "--name", "myname", "--checksum", "sha256:123", "--scantype", "docker", "--outputfile", "outfile" };
```

Comment:
`sha256:123` -> should throw an exception, because `123` is not a sha256 checksum (too short).